src/handlers/user-start-stop.ts

```diff
@@ -33,6 +33,12 @@ export async function userSelfAssign(context: Context<"issues.assigned">): Promi
   const { issue } = payload;
   const deadline = getDeadline(issue.labels);
 
+  // We avoid posting a message if the bot is the actor to avoid double posting
+  if (payload.sender.type === "Bot") {
```

Comment:
You could merge with the `if` below since they are the same response

Reply:
Will do so and merge.